python/tk_maya/panel_generation.py

```diff
@@ -264,18 +264,16 @@ def build_workspace_control_ui(shotgun_panel_name):
     """
 
     from maya.OpenMayaUI import MQtUtil
+    import sgtk.platform
+    from sgtk.platform.qt import shiboken
 
     # In the context of this function, we know that we are running in Maya 2017 and later
     # with the newer versions of PySide and shiboken.
     # Try importing PySide6 modules first
     try:
         from PySide6 import QtWidgets
-        import shiboken6 as shiboken
     except ImportError:
         from PySide2 import QtWidgets
```

Comment:
Why not using the same strategy here? 
```python
from sgtk.platform.qt import QtWidgets
```

Or something similar?

Reply:
Investigating a bit this part it's exactly what @staceyoue previously mentioned. In `sgtk.platform.qt` we are patching PySide6 to conform to the PySide interface, so all the content from the `QtWidgets` module was moved to `QtGui` in the [pyside6_patcher.py](https://github.com/shotgunsoftware/tk-core/blob/fe3d8d5375ef912eabf35fd938b078fd7b44f588/python/tank/util/pyside6_patcher.py#L409) file, since they belong there in PySide.

So, I believe to implement your strategy, we should be able to import it using `from sgtk.platform.qt import QtGui` and then change every reference of `QtWidgets` to `QtGui`.